cdci_data_analysis/analysis/instrument.py

```diff
@@ -63,6 +63,22 @@
 # TODO: this is not preserved between requests, and is not thread safe. Why not pass it in class instances?
 params_not_to_be_included = ['user_catalog',]
 
+non_parameter_args = ['instrument', 
+                      'query_status', 
+                      'query_type', 
+                      'product_type', 
+                      'session_id', 
+                      'token',
+                      'api',
+                      'oda_api_version',
+                      'off_line',
+                      'job_id',
+                      'async_dispatcher',
+                      'allow_unknown_args']
+# NOTE: arguments are passed in the request to the dispatcher
+# some arguments are used to set the values of the analysis parameters
+# the parameter is a subclass of Parameter and may use several arguments to set it's value 
```

Comment:
cool, but there is a bit more to it.
something should be said about Parameter (and their subclasses) instances,which have names.
Notable that several parameters, even built from from different classes may use some of the same arguments.